packages/communication-react/review/communication-react.api.md

```diff
@@ -836,7 +836,6 @@ export const COMPOSITE_LOCALE_ZH_TW: CompositeLocale;
 
 // @public
 export const COMPOSITE_ONLY_ICONS: {
-    failedToJoinCallDueToNoNetworkIcon: JSX.Element;
```

Comment:
This API file was generated incorrectly in the PR below this one. This diff will disappear once that lands and I merge it into this PR.